dotnet/src/Connectors/Connectors.AssemblyAI.UnitTests/Services/AssemblyAIAudioToTextServiceTests.cs

```diff
@@ -126,9 +266,9 @@ public async Task HttpErrorShouldThrowWithErrorMessageAsync()
         ];
 
         // Act & Assert
-        await Assert.ThrowsAsync<HttpOperationException>(
+        await Assert.ThrowsAsync<ApiException>(
```

Comment:
Exposing custom exception from AssemblyAI is rather bad idea. Even OpenAI connector uses translation to HttpOperationException. You eventually can use in very specific cases KernelException with embedded ApiException.

Reply:
Done!